src/inspector_agent.cc

```diff
@@ -338,11 +345,58 @@ AgentImpl::~AgentImpl() {
   data_written_ = nullptr;
 }
 
+void InspectorCallSequently(const v8::FunctionCallbackInfo<v8::Value>& args) {
```

Comment:
This is not just calling sequentially now. Perhaps, wrapConsoleCall or something similar would be a better name.